### DISCUSSION TITLE
Something about GNSS data from UNR MAGNET

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#493] [General]
I downloaded GNSS time series data in tenv3 format from this website. When I observed the data of G100 site, I found that the value of N direction was obviously large. May I ask how to use enu value?
![GCZ 3~M`XG1ZF%9474F74WV](https://user-images.githubusercontent.com/112800803/196379630-1154ab5d-3338-4f2d-ad4b-79276146f6e0.png)

**Comment by Xiaohua-Eric-Xu** [comment]
Depend on what the station is measuring, 2.5mm in two weeks is not surprisingly large.
